messenger.rst

```diff
@@ -587,6 +587,28 @@ to handle messages in a priority order:
 The worker will always first look for messages waiting on ``async_priority_high``. If
 there are none, *then* it will consume messages from ``async_priority_low``.
 
+.. _messenger-limit-queues:
+
+Limit Consuming to Specific Queues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.3
+
+    Limiting the worker to specific queues was introduced in Symfony 5.3.
+
+Some transports (notably AMQP) have the concept of exchanges and queues. A Symfony
+transport is always bound to an exchange. By default, the worker consumes from all
+queues attached to the exchange of the specified transport. However, there are use
+cases to want a worker to only consume from specific queues.
+
+You can limit the worker to only process messages from specific queues:
+
+.. code-block:: terminal
+
+    $ php bin/console messenger:consume my_transport --queues=fasttrack
+
+To allow using the `queues` option, the receiver must implement the `QueueReceiverInterface`.
```

Comment:
```suggestion
To allow using the ``queues`` option, the receiver must implement the ``QueueReceiverInterface``.
```

Reply:
oh, its been a while that i worked on .rst and not markdown :man_facepalming: